apps/radix-ssr-testing/server.ts

```diff
@@ -1,5 +1,5 @@
 import { APP_BASE_HREF } from '@angular/common';
-import { CommonEngine } from '@angular/ssr';
+import { CommonEngine } from '@angular/ssr/node';
```

Comment:
I am just curious - I've seen that article and regarding the ssr files migration https://medium.com/@captainDuckay/angular-ssr-19-upgrade-20c80aacf95e I might be useful or it might not ;) I am only bringing up with what I've found before :)

Reply:
Thanks, I still need to dedicate some time to testing for SSR separately :)